binaries/cuprated/src/rpc/rpc_handler.rs

```diff
@@ -167,14 +167,9 @@ pub struct CupratedRpcHandler {
     /// Handle to the blockchain context service.
     pub blockchain_context: BlockchainContextService,
 
-    /// Handle to the blockchain manager.
-    pub blockchain_manager: BlockchainManagerHandle,
-
     /// Read handle to the transaction pool database.
     pub txpool_read: TxpoolReadHandle,
-
-    /// TODO: handle to txpool service.
-    pub txpool_manager: std::convert::Infallible,
+    // TODO: handle to txpool service.
```

Comment:
These need to be reworked to use the new APIs.

A lot of the diffs in this PR are `&mut state.blockchain_manager` -> `todo!()`.